Update 'User Sessions' chapter with new Admin UI (#1538)
keycloak/keycloak-documentation: server_admin/topics/sessions/revocation.adoc

[[_revocation-policy]]

=== Revoking active sessions
[role="_abstract"]

If your system is compromised, you can revoke all active sessions and access tokens.

.Procedure
. Click *Sessions* in the menu.
ifeval::[{project_community}==true]
. From the *Actions* list, select *Sign out all active sessions*.
endif::[]
ifeval::[{project_product}==true]
. Click the *Revocation* tab.
endif::[]

.Revocation
image:{project_images}/revocation.png[Revocation]

> . Specify a time and date where sessions or tokens issued before that time and date are invalid using this console. 

Reviewer: When you do this, do you see the screen shown.  It may not apply unless I actually have other active sessions, but for me it just ends my sessions.

Reviewer: @jonkoops The PR looks good, but I am not seeing what you are seeing.  I just get logged out.  I assume it is because I have only one active session. Does that make sense?

Author: Yes, that is correct. Since the admin session is also terminated this is what will happen.

> === Revocation policies

Reviewer: ```suggestion
=== Revoking active sessions
```

[role="_abstract"]

> ifeval::[{project_community}==true]
> If your system is compromised, you can revoke all active sessions and access tokens by clicking the `Sessions` screen's *Revocation* button in the 'Action' menu.
> endif::[]
> ifeval::[{project_product}==true]
> If your system is compromised, you can revoke all active sessions and access tokens by clicking the `Sessions` screen *Revocation* tab.
> endif::[]
> 
> .Revocation
> image:{project_images}/revocation.png[A screenshot showing the 'Revocation' functionality.]

Reviewer: ```suggestion
If your system is compromised, you can revoke all active sessions and access tokens.

.Procedure
. Click *Sessions* in the menu.
ifeval::[{project_community}==true]
. From the *Actions* list, select *Sign out all active sessions*.
endif::[]
ifeval::[{project_product}==true]
. Click the *Revocation* tab.
endif::[]
+
.Revocation
image:{project_images}/revocation.png[Revocation]
```

> You can specify a time and date where sessions or tokens issued before that time and date are invalid using this console. Click *Set to now* to set the policy to the current time and date. Click *Push* to push this revocation policy to any registered OIDC client with the {project_name} OIDC client adapter.

Reviewer: ```suggestion
. Specify a time and date where sessions or tokens issued before that time and date are invalid using this console. 
* Click *Set to now* to set the policy to the current time and date. 
* Click *Push* to push this revocation policy to any registered OIDC client with the {project_name} OIDC client adapter.
```
